xpore/scripts/dataprep.py

```diff
@@ -326,43 +339,44 @@ def parallel_preprocess_gene(eventalign_filepath,fasta_dict,annotation_dict,is_g
 #             continue
 #         else:
 #             gene_ids = gene_ids.union([g_id])
-            
 
     # Load tasks into task_queue.    
     gene_ids_processed = []
 
     with open(eventalign_filepath,'r') as eventalign_result:
-
         for gene_id in g2t_mapping:
-                        
+            # if resuming, skip genes that have already been processed
```

Comment:
please keep this comment

Reply:
?
I added that comment, to make the intention of the continue clear